src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+
+    <title>Product cards</title>
+
+    <link
+      rel="stylesheet"
+      href="./styles/index.css"
+    >
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__container">
+        <img src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        class="product-picture"
+        >
+        <h2 class="product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="product-code">Product code: 195434</p>
```

Comment:
use card as a block for naming classes to follow BEM 
```suggestion
        <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
        <p class="card__code">Product code: 195434</p>
```